src/utils/env.js

```diff
@@ -20,7 +20,11 @@ const DeviceParser = [
   [[UAParser.DEVICE.TYPE, UAParser.DEVICE.SMARTTV]]
 ];
 
-let Env = new UAParser(undefined, {browser: SAMSUNGBrowserParser, device: DeviceParser, os: LGOSParser}).getResult();
+const EdgeChromium = [[/(edg)\/((\d+)?[\w.]+)/i], [[UAParser.BROWSER.NAME, 'Edge'], UAParser.BROWSER.VERSION, UAParser.BROWSER.MAJOR]];
+
+const browserDetect = EdgeChromium.concat(SAMSUNGBrowserParser);
```

Comment:
is browserDetect the best naming for this variable? maybe something like samsungOrChromiumDetect?

Reply:
I don't think so, according to your suggestion next time it'll be samsungOrChromiumSomethingElseDetect and after that.. bottom line you don't want to create infinite variable names which will be meaningless,  I used the the name of the attribute that I want to parse.

Reply:
ok, so maybe just uppercase cause its a constant

Reply:
About the constant, it's `Edge` cause in chromium it wrote `Edg` in user agent and in other cases is `Edge`.

Reply:
my bad, I am used that constants are uppercased. But not in Kaltura :)